CppSamples/EditData/EditGeodatabaseWithTransactions/EditGeodatabaseWithTransactions.cpp

```diff
@@ -0,0 +1,562 @@
+// [WriteFile Name=EditGeodatabaseWithTransactions, Category=EditData]
+// [Legal]
+// Copyright 2025 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#ifdef PCH_BUILD
+#include "pch.hpp"
+#endif // PCH_BUILD
+
+// sample headers
+#include "EditGeodatabaseWithTransactions.h"
+
+// ArcGIS Maps SDK headers
+#include "ErrorException.h"
+#include "Feature.h"
+#include "FeatureLayer.h"
+#include "FeatureTemplate.h"
+#include "FeatureType.h"
+#include "Geodatabase.h"
+#include "GeodatabaseFeatureTable.h"
+#include "Geometry.h"
+#include "GeometryEngine.h"
+#include "Graphic.h"
+#include "GraphicListModel.h"
+#include "GraphicsOverlay.h"
+#include "GraphicsOverlayListModel.h"
+#include "LayerListModel.h"
+#include "Map.h"
+#include "MapQuickView.h"
+#include "MapTypes.h"
+#include "Point.h"
+#include "SimpleLineSymbol.h"
+#include "SpatialReference.h"
+#include "SymbolTypes.h"
+#include "Viewpoint.h"
+
+// Qt headers
+#include <QException>
+#include <QFuture>
+#include <QFutureWatcher>
+#include <QStandardPaths>
+#include <QTimer>
+
+using namespace Esri::ArcGISRuntime;
+
+// helper method to get cross platform data path
+namespace
+{
+  QString defaultDataPath()
+  {
+    QString dataPath;
+
+#ifdef Q_OS_IOS
+    dataPath = QStandardPaths::writableLocation(QStandardPaths::DocumentsLocation);
+#else
+    dataPath = QStandardPaths::writableLocation(QStandardPaths::HomeLocation);
+#endif
+
+    return dataPath;
```

Comment:
I recall this is copied from other samples, but I just noticed it can be simplified. Optional. Not terrible the way it is.


```suggestion
#ifdef Q_OS_IOS
    return QStandardPaths::writableLocation(QStandardPaths::DocumentsLocation);
#else
    return QStandardPaths::writableLocation(QStandardPaths::HomeLocation);
#endif
```